defaults.env

```diff
@@ -9,6 +9,8 @@ PORT=8537
 BASE_URL=http://localhost:8537
 DATABASE_MAXCONN=10
 
+CRYPTO_KEYS="1YrzmaGBUeFrwD9SpBOqv33a2ElGns9mUtldAmoU7hs="
```

Comment:
Maybe leave a comment here saying that extra keys should be separated by a space?